advocacy_docs/edb-postgres-ai/migration-etl/data-migration-service/supported_versions.mdx

```diff
@@ -51,6 +52,16 @@ Data Migration Service supports migrating to both HCP-managed databases (on clus
 !!!note 
    If the destination is an HCP-managed database, AWS must be the provider for the Hybrid Control Plane cluster. Other Cloud Service providers are currently not supported.
 
+### Oracle 
+
+EDB doesn't support migrations to Oracle destinations.
```

Comment:
We do support migrations to Oracle destinations, particularly for fallback scenarios.  We are planning to make this capability available later in the year.  So, I would suggest rewording this sentence to:

> The Data Migration Service does not yet support migrations to Oracle destinations.